python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -427,7 +441,7 @@ async def _process_function_call(
         arguments: "KernelArguments",
         function_call_count: int,
         request_index: int,
-        function_call_behavior: FunctionCallBehavior,
+        function_call_behavior: FunctionChoiceBehavior | FunctionCallBehavior,
```

Comment:
Perhaps:
```suggestion
        function_choice_behavior: FunctionChoiceBehavior | FunctionCallBehavior,
```
now that FunctionCallBehavior is deprecated?

Reply:
I wish, but we cannot rename this parameter name as it would breaking functionality for existing users who are implementing this on their own. I am aware of some customers who do handle tool calls manually and if we changed this parameter name, their code will break.